cirq-core/cirq/sim/simulator.py

```diff
@@ -157,6 +157,9 @@ def compute_amplitudes(
                 as an integer array where each integer is formed from measured
                 qubit values according to `qubit_order` from most to least
                 significant qubit, i.e. in big-endian ordering.
+                Binary literals should be input with the prefix 0b or 0B. 
+	        For example, 0010 should be input as 0b0010 or 0B0010.
```

Comment:
As indicated by the function signature, the bitstrings are supposed to be good old python ints. All valid ways of writing int literals will work, not just 0b-prefixed binary literals.